test/razorpay/test_dispute.rb

```diff
@@ -0,0 +1,68 @@
+require 'test_helper'
+
+module Razorpay
+
+  class RazorpayDisputeonTest < Minitest::Test
+    class Dispute < Razorpay::Entity; end
+
+    def setup
+      @dispute_id = 'disp_XXXXXXXXXXXXX'
+
+      # Any request that ends with disputes/dispute_id
+      stub_get(%r{disputes\/#{Regexp.quote(@dispute_id)}$}, 'fake_dispute')
+    end
+
+    def test_dispute_should_be_defined
+      refute_nil Razorpay::Addon
+    end
+
+    def test_addon_should_be_available
+      dispute = Razorpay::Dispute.fetch(@dispute_id)
+      assert_instance_of Razorpay::Dispute, dispute, 'Dispute not an instance of Dispute class'
+      assert_equal @dispute_id, dispute.id
+      assert_equal 'INR', dispute.currency
+    end
+
+    def test_fetch_all_dispute
+      stub_get(/disputes$/, 'dispute_collection')
+      dispute = Razorpay::Dispute.all
+      assert_instance_of Razorpay::Collection, dispute, 'Dispute should be an array'
+      refute_empty dispute.items, 'Dispute should be more than one'
+    end
+
+    def test_dispute_accept
```

Comment:
please add test cases for failure cases of these APIs.

Reply:
Added failure test cases